src/profiler/telemeh.c

```diff
@@ -314,7 +312,7 @@ MVM_PUBLIC void MVM_telemetry_init(FILE *outfile)
 
     beginningEpoch = epochRecord->u.epoch.time;
 
-    threadCreateError = uv_thread_create(&backgroundSerializationThread, (uv_thread_cb)backgroundSerialization, (void *)outfile);
+    threadCreateError = uv_thread_create((uv_thread_t *)&backgroundSerializationThread, backgroundSerialization, (void *)outfile);
```

Comment:
Why the type cast to `uv_thread_t *`? `backgroundSerializationThread` seems to already be a `uv_thread_t`.

Reply:
With `gcc (Ubuntu 9.2.1-9ubuntu2) 9.2.1 20191008` I otherwise get the warning:
```
src/profiler/telemeh.c: In function ‘MVM_telemetry_init’:
src/profiler/telemeh.c:317:74: warning: cast between incompatible function types from ‘void * (*)(void *)’ to ‘void (*)(void *)’ [-Wcast-function-type]
  317 |     threadCreateError = uv_thread_create(&backgroundSerializationThread, (uv_thread_cb)backgroundSerialization, (void *)outfile);
      |                                                                          ^
```